api/rest/client/client.go

```diff
@@ -39,28 +39,28 @@ type Config struct {
 	Password string
 
 	// The ipfs-cluster REST API endpoint in multiaddress form
-	// (takes precedence over host:port). Only valid without PeerAddr.
+	// (takes precedence over host:port). It this address contains
+	// an /ipfs/, /p2p/ or /dnsaddr, the API will be contacted
+	// through a libp2p tunnel, thus getting encryption for
+	// free. The latter overrides any SSL configurations.
```

Comment:
> The latter overrides any SSL configurations.

I am not sure what is going to override any SSL configurations? On first read, I thought `latter` was referring to `/dnsaddr` but that doesn't make much sense.